src/graphql/typeDefs/mutations/sessions.graphql

```diff
@@ -1,5 +1,6 @@
 type SessionsMutation {
-  create(session: SessionCreateInput!): Session! @auth(requires: "sessions")
+  create(eventId: ID!, session: SessionCreateInput!): Session!
+    @auth(requires: "sessions")
   delete(id: ID!): ID @auth(requires: "sessions")
```

Comment:
We sure we want a user to have the permissions to delete a session? What if it happens after accepted?

Reply:
prolly not. or maybe just delete their own if not accepted.

Reply:
it's also just a stub.